Servers/UI/OJS.Servers.Ui/ClientApp/src/hooks/submissions/use-submissions.tsx

```diff
@@ -123,16 +124,16 @@ const SubmissionsProvider = ({ children }: ISubmissionsProviderProps) => {
 
     useEffect(
         () => {
-            if (!isNil(submitCodeResult) && submitCodeResult.status !== 200) {
-                setSubmitMessage(submitCodeResult.detail);
+            if (!isNil(submitCodeResult) && submitCodeResponse?.status !== 200) {
+                setSubmitMessage('error');
```

Comment:
Is this correct - setSubmitMessage('error')
Is the old implementation invalid - setSubmitMessage(submitCodeResult.detail)

Reply:
Neither the old nor the hardcoded value were correct. I've updated the implementation.